logging/README.md

```diff
@@ -0,0 +1,3 @@
+# Logging
+
+## Scripts to manage the Elastic config 
```

Comment:
Would be nice to add:
- What is each one for specifically?
- How (and when) do we run them?

Reply:
Nice spot, definitely a half formed doc. Updated - what do you think?